canopeum_backend/pyproject.toml

```diff
@@ -3,17 +3,26 @@ name = "Canopeum"
 version = "0.0.0"
 requires-python = ">= 3.12"
 dependencies = [
-  "Django>=5.0.8", # Fix for https://github.com/BesLogic/releaf-canopeum/security/dependabot/25
+  "Django>=5.0.8",
+  # Fix for https://github.com/BesLogic/releaf-canopeum/security/dependabot/25
   "dj-database-url",
   "django-cors-headers",
   "djangorestframework-camel-case",
-  "djangorestframework-simplejwt", # TODO: Bump above 5.3.1 as soon as fixed for https://github.com/BesLogic/releaf-canopeum/security/dependabot/3
-  "djangorestframework>=3.15.2", # Fix for https://github.com/BesLogic/releaf-canopeum/security/dependabot/15
+  "djangorestframework-simplejwt",
+  # TODO: Bump above 5.3.1 as soon as fixed for https://github.com/BesLogic/releaf-canopeum/security/dependabot/3
+  "djangorestframework>=3.15.2",
+  # Fix for https://github.com/BesLogic/releaf-canopeum/security/dependabot/15
   "drf-spectacular-sidecar",
-  "drf-spectacular>=0.27.2", # https://github.com/tfranzel/drf-spectacular/pull/1216
+  "drf-spectacular>=0.27.2",
+  # https://github.com/tfranzel/drf-spectacular/pull/1216
   "googlemaps",
   "mysqlclient",
   "python-dotenv",
+  "openmeteo-requests>=1.3.0",
+  "requests-cache>=1.2.1",
+  "retry-requests>=2.0.0",
+  "numpy>=2.1.0",
+  "pandas>=2.2.2",
```

Comment:
I don't see either being directly used in our code
```suggestion
```

Reply:
Yes you're true, I followed the docs in the dark 🧑‍🦯